aten/src/ATen/native/cuda/linalg/BatchLinearAlgebra.cpp

```diff
@@ -2745,7 +2744,7 @@ void lstsq_kernel(const Tensor& a, Tensor& b, Tensor& /*rank*/, Tensor& /*singul
         "Please rebuild with cuSOLVER.");
 #endif
   } else { // m >= n
-//#if !AT_ROCM_ENABLED()
+#if !AT_ROCM_ENABLED()
```

Comment:
I'm not sure we want to add this. I believe gels are implemented in hipsolver so we should be using that instead of magma

Reply:
This code matches upstream, so I believe we need a separate commit to remove them after confirming gels are implemented in hipSolver.
ref - https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/cuda/linalg/BatchLinearAlgebra.cpp#L2747